Jenkinsfile-Release

```diff
@@ -0,0 +1,249 @@
+/*
```

Comment:
this whole file is a copy of the original Jenkinsfile that used to do the release. It retains the guards that still allow it to only release from "master". I didn't see the need to remove those checks yet, but I could if reviewers insist.